_i18n/en/resources/user-guides/restore_account.md

```diff
@@ -1,53 +1,131 @@
+### What is needed to restore a wallet from mnemonic seed?
 
-- Press enter again and you will be prompted for the 25 word electrum style mnemonic seed that you retrieved earlier
+In order to restore a wallet from mnemonic seed, you must have a valid Monero mnemonic seed. A Monero mnemonic seed is a phrase with 13, 14, or 25 random words (multiple languages are supported). There are three types of Monero mnemonic seed:
 
--  You will then be prompted with "Restore from specific blockchain height (optional, default 0):" Default will start the restore process from the beginning of the Monero blockchain. If you don't know the specific blockchain height, just hit enter. (Specifying a specific blockchain height will start the restore process from that specific height. This will save a bit of time in scanning, if you know what starting blockchain height your initial funds were transacted for this specific account.)
+- **25 words mnemonic seed**: used by most Monero wallets and supported by all Monero wallets.
+- **14 words mnemonic seed with embedded restore height**: used and supported by Feather wallet.
+- **13 words mnemonic seed**: deprecated, previously used by MyMonero wallet. Supported by most Monero wallets.
 
-After you have entered the 25 word mnemonic seed and have chosen your specific blockchain height, monero-wallet-cli will generate the same public address and view key as your old wallet and begin the refresh process automatically. (Please be patient as the refresh process may take a while.)
+Your mnemonic seed has all the information needed to restore your wallet.
 
-### Account Software:  monero-wallet-gui
+It is useful to know your wallet restore height or the creation date of your wallet, since it makes your wallet synchronization faster.
 
-Lauch `monero-wallet-gui`. If this is the first time you launched it go to the next step, otherwise click `Cancel`:
+### Where can I find my mnemonic seed?
 
-![cancel opening](/img/resources/user-guides/en/restore_account/cancel-opening.png)
+Most Monero wallets ask the user to write down a mnemonic seed on a paper backup when a wallet is being created. So it's up to you to remember where and how you stored your mnemonic seed.
 
-Select your appropriate language `English(US)`:
+Alternatively, if you have access to the wallet being restored, you can find your mnemonic seed in the following locations:
 
-![choose language](/img/resources/user-guides/en/restore_account/choose-language.png)
+- **Monero GUI**: go to `Settings` > `Wallet` and open `Show seed & keys`.
+- **Monero CLI**: use the command `seed`.
+- **Cake Wallet**: open `Show seed/keys` on the main menu.
+- **Feather**: on the top menu click on `Wallet` > `Seed`.
+- **Monerujo**: open `Show Secrets!` on the main menu.
+- **MyMonero**: open your wallet and click on `Address`. Your mnemonic seed is shown as `Secret Mnemonic`.
+- **Edge**: long press the XMR wallet on the main page and open `Master Private Key`.
```

Comment:
I don't think we should add instructions for so many wallets. Only the GUI and CLI will do it. We don't need to provide instructions for every wallet we know. These guide will also need to be maintained.